### PR TITLE
Add blog-app views

### PR DESCRIPTION
**Pull request by EzekielUtshudi**
### **In this PR :** 
 

- [ ] Add blog-app views as implemented by the wireframe

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,29 +1,5 @@
...
-    <h3>Username</h3>
-    <h4>Number of posts: X</h4>
-  </div>
-</div>
\ No newline at end of file
+<h1>List of all users</h1>
+
+<% @user.each do |user| %>
+    <%= render user %>
+<% end %>
```
Comment (medaminedev66) [inline app/views/users/index.html.erb]: - [ ] Following the project requirements, the index view page of users should match the given [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=ABIFLA3WTIGWG4Z5XBP2BQLBGXZKY)
- [ ] Please make sure the other pages match the given wireframe as well.

-  Your design:
image


- The wireframe:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @EzekielUtshudi  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

## Required Changes ♻️
- [ ] Please add one or two users using seed data and also give the command in README file `rails db:seed` to load seed data into the local to test all the post, comment, and like features.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (EzekielUtshudi) [reply app/views/users/index.html.erb]: Implement the design from the sneak peek wireframes.
Styling is nice to have but focus on making sure everything functions first. @medaminedev66 

please I focused more on the function as it is for me a complex language and I am running windows, can you assist?

Comment (EzekielUtshudi) [reply app/views/users/index.html.erb]: image

Comment (KDlamini) [reply app/views/users/index.html.erb]: Hi @EzekielUtshudi,

What the requirement means is that the layout of your elements should be the same as the wireframe, yes no styling is required at the moment, but the layout should be the same. This means:

- The image should be on the left container, and the author's name and the number of posts should be on the right container.
- You might need to still use a little bit of CSS to align your items like Flexbox to align items in a row.

Comment (EzekielUtshudi) [reply app/views/users/index.html.erb]: hey @KDlamini

Comment (EzekielUtshudi) [reply app/views/users/index.html.erb]: can that be optional changes that I can do on the next milestone of the project of forms ? @KDlamini

**Review by KDlamini — APPROVED** [review]
### Hi @EzekielUtshudi 🙋‍♀️,  

Good job! 
You have done a great job implementing all the required changes. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Great PR title
✔️ Correct use of GitFlow
✔️ No linter errors

# Status: Approved ✔️ 
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**